frontend/src/components/SearchResultForMobile.js

```diff
@@ -0,0 +1,93 @@
+import { Col, Row } from 'react-bootstrap';
+import defaultImage from '../images/defaultimage.png';
+import '../styles/SearchResultForMobile.css';
+import { IoIosAlarm } from 'react-icons/io';
+import { IoBarbellSharp } from 'react-icons/io5';
+import { useEffect, useState } from 'react';
+import { getRecipesWithMatchingIngredients } from './apiCalls';
+
+function SearchResultForMobile({ ingredients, searchBy }) {
+  const [recipe, setRecipe] = useState([{}]);
+
+  useEffect(() => {
+    const getRecipes = async () => {
+      const recipeList = await getRecipesWithMatchingIngredients(
+        ingredients,
+        searchBy
+      );
+      recipeList.sort((a, b) => a.name.localeCompare(b.name));
+      setRecipe(recipeList);
+    };
+    getRecipes();
+  }, [ingredients, searchBy]);
+
+  return (
+    <div>
+      <Row>
+        <Col></Col>
+        <Col>
+          {recipe
+            ? recipe.map((r) => (
+                <div className="contForMobile">
+                  <img
+                    src={
+                      r.image
+                        ? r.image.fileName === 'abc'
+                          ? defaultImage
+                          : `/api/image/${r.image.id}`
+                        : defaultImage
+                    }
+                    alt="KÉP HELYE"
+                  />
+                  <div className="contForMobile__text">
+                    <div className="recipeNameForMobile">
+                      {r.name ? r.name.substring(0, 20) : 'loading'}
+                    </div>
+                    <div className="contForMobile__text__star">
+                      <span className="fa fa-star checked"></span>
+                      <span className="fa fa-star checked"></span>
+                      <span className="fa fa-star checked"></span>
+                      <span className="fa fa-star checked"></span>
+                      <span className="fa fa-star checked"></span>
+                    </div>
+                    <p className="description">
+                      {' '}
```

Comment:
Ez a space kell?

Reply:
A prettier illeszti be ha akarjuk ha nem